src/main/kotlin/inc/evil/d17map/dtos/ClassroomRequest.kt

```diff
@@ -17,5 +16,6 @@ data class ClassroomRequest(
     val capacity: Int,
 
     val equipmentIds: Set<UUID>,
-    val floor: Floor
+    val floorName: String,
```

Comment:
Czy floorName będzie np. "1" w przypadku gdy mówimy o pierwszym piętrze? Rozumiem ideę, że mogą istnieć budynki z piętrami np. "A1", bo czemu by nie, tylko się zastanawiam, czy w takim wypadku nie lepiej by było to pole nazwać `floorNumber`, samo `floor`, a może w ogóle `level`?

Osobiście bym nazwę pola dał albo na `floor` albo na `level`, a typ zostawił jak jest, ale można też zapytać co myśli front

Reply:
moim zdaniem jest ok bo sugeruje dokładnie w stu procentach to,  co dostajemy - w przypadku floor oczekiwałabym **_obiektu_** typu floor, natomiast level wprowadzałoby niejednoznaczność - czym jest floor a czym level, czy to to samo?  czy może jeszcze coś innego? 

Nie widzę sensu, żeby tutaj jakoś dodawać jeszcze kolejne nazwy i potem zastanawiać się co jest czym.

Reply:
Ok